.github/workflows/test.yml

```diff
@@ -56,6 +64,34 @@ jobs:
 
     - name: Tox tests
       run: |
+        # setup a redis sentinel
```

Comment:
you can't specify a command to a service as of now and this needs to be on the host network since redis does not use DNS names but IPs only. For the master this isn't required because clients will ignorantly connect to 127.0.0.1 but because of the port specification that will just work. For the sentinel 127.0.0.1 is itself which is not correct and it needs to be on the host network so that it gets the same port forwarding the client gets.